src/openms/include/OpenMS/METADATA/PeptideIdentification.h

```diff
@@ -166,6 +167,28 @@ namespace OpenMS
     /// returns all peptide hits which reference to a given protein accession (i.e. filter by protein accession)
     static std::vector<PeptideHit> getReferencingHits(const std::vector<PeptideHit>&, const std::set<String>& accession);
 
+      /**
+      @brief Builds MultiMap over UID and pair of index of ConsensusFeature in ConsensusMap and index of PeptideIdentification
+
+      @param cmap is used to extract the ConsensusFeatures and their PeptideIdentifications and creates identifier_tomsrunpath out of it
```

Comment:
`@param cmap All PI's of the CMap are enumerated and their UID -> pair mapping is computed`